lib/Util/PHPUtil.php

```diff
@@ -99,7 +99,7 @@ public function chmodWrapper($fileName, $mode) {
         if (chmod($fileName, $mode)) {
             return true;
         } else {
-            throw new NotFoundException($this->l10n->t('Cannot set permissions to temporary file for tesseract.'));
+            throw new NotFoundException($this->l10n->t('Cannot set set permissions for temporary Tesseract file.'));
```

Comment:
That's right, @comradekingu Could you change this? Then I would merge it. Thanks!